extensions/micrometer/runtime/src/main/java/io/quarkus/micrometer/runtime/export/PrometheusMeterRegistryProvider.java

```diff
@@ -58,7 +58,7 @@ public CollectorRegistry collectorRegistry() {
 
     @Produces
     @Singleton
-    @AlternativePriority(Interceptor.Priority.APPLICATION + 100)
+    @AlternativePriority(Interceptor.Priority.APPLICATION)
```

Comment:
Is it the correct priority for a framework bean?

Also, I wonder if we should make it work another way:
- use `@DefaultBean`
- have a qualifier with a name to define others
?

The current setup looks very weird to me. @mkouba could you have a look at this and the discussion here: https://github.com/quarkusio/quarkus/pull/15219 ?

Reply:
There is a valid use case to use both the bean automatically configured by the runtime AND an additional, custom bean configured manually, which is why I did not use a DefaultBean to begin with.
 
I am trying to make a value that is more obvious to supersede, if desired (though I don't think most people will(

Reply:
I need to think on it a bit more, but using DefaultBean and auto-ejecting the framework defined bean, while perhaps more obvious on the surface, isn't the right answer.. but the @AlternativePriority isn't consistently selecting/omitting the way I expected.